src/Uno.Templates/content/unoapp/solution-config.props.sample

```diff
@@ -12,11 +12,11 @@
     <PropertyGroup>
         <!-- Uncomment each line for each platform that you want to build: -->
```

Comment:
Maybe this could be something along the lines of "If you want to build specific platforms only, uncomment them below" (as by default all platforms are built)

Reply:
Yep, we'll add more info here